experimental/datacommons-js/README.md

```diff
@@ -0,0 +1,351 @@
+```
+
+## Examples
+
+For a complete example, see [example.html](./example.html)
+
+### Bar chart
+
+Draw a multi-variable bar chart
+
+Initialize empty div
+
+```html
+<div id="bar-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawBar(document.getElementById("bar-example"), {
+  id: "bar-chart-1",
+  svgChartHeight: 200,
+  className: "my-chart",
+  apiRoot: datacommons.root,
+  title:
+    "Population Below Poverty Level Status in Past Year in States of United States (2020)",
+  place: {
+    dcid: "country/USA",
+    name: "United States of America",
+    types: ["Country"],
+  },
+  enclosedPlaceType: "State",
+  statVarSpec: [
+    {
+      name: "Population Below Poverty Level Status in Past Year",
+      statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+    },
+  ],
+});
+```
+
+Renders:
+
+![Bar chart](./assets/bar-chart.png "Bar Chart")
+
+### Line chart
+
+Draw a multi-variable line chart
+
+Initialize empty div
+
+```html
+<div id="line-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawLine(document.getElementById("line-example"), {
+  id: "line-chart-1",
+  svgChartHeight: 200,
+  className: "my-chart",
+  apiRoot: datacommons.root,
+  title:
+    "Population Below Poverty Level Status in Past Year in United States (2020)",
+  place: {
+    dcid: "country/USA",
+    name: "United States of America",
+    types: ["Country"],
+  },
+  statVarSpec: [
+    {
+      name: "Population Below Poverty Level Status in Past Year",
+      statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+    },
+  ],
+});
+```
+
+Renders:
+![Line chart](./assets/line-chart.png "Line Chart")
+
+### Map chart
+
+Draw a choropleth map
+
+Initialize empty div
+
+```html
+<div id="map-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawMap(document.getElementById("map-example"), {
+  id: "map-chart-1",
+  svgChartHeight: 200,
+  className: "my-chart",
+  apiRoot: datacommons.root,
+  title:
+    "Population Below Poverty Level Status in Past Year in States of United States (2020)",
+  place: {
+    dcid: "country/USA",
+    name: "United States of America",
+    types: ["Country"],
+  },
+  enclosedPlaceType: "State",
+  statVarSpec: {
+    name: "Population Below Poverty Level Status in Past Year",
+    statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+  },
+});
+```
+
+Renders:
+
+![Map chart](./assets/map-chart.png "Map Chart")
+
+### Ranking chart
+
+Draw ranking chart (bar chart + ordered table)
+
+Initialize empty div
+
+```html
+<div id="ranking-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawRanking(document.getElementById("ranking-example"), {
+  placeName: "USA",
+  placeType: "State",
+  withinPlace: "country/USA",
+  statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+  isPerCapita: false,
+  scaling: 1,
+  unit: "",
+  date: "2020",
+});
+```
+
+Renders:
+
+![Ranking chart](./assets/ranking-chart.png "Ranking Chart")
+
+## API
```

Comment:
(for future PR), maybe not working on the API spec until they are fully determined so save some time.

Reply:
sg!